dynamic-plugins.default.yaml

```diff
@@ -823,17 +823,23 @@ plugins:
                   props:
                     width: 1500
                     height: 800
+
   - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment
-    disabled: true
     pluginConfig:
       dynamicPlugins:
         frontend:
           janus-idp.backstage-plugin-analytics-provider-segment:
             apiFactories:
               - importName: SegmentAnalyticsApi
       app:
+        backend:
+          csp:
+            script-src:
+              - '''self'''
+              - '''unsafe-eval'''
+              - 'https://cdn.segment.com'
         analytics:
           segment:
             writeKey: ${SEGMENT_WRITE_KEY}
-            maskIP: ${SEGMENT_MASK_IP} # prevents IP addresses from being sent if true
+            maskIP: true # prevents IP addresses from being sent if true
```

Comment:
The default value is controlled by the plugin, so it can change independently from this. We can change this in the plugin, but I would keep it here hardcoded like this.